### PR TITLE
docs: Improve migration instructions

### PR DESCRIPTION
**Pull request by Luc1412**
While testing the migration, with the deb package, I encountered a few problems. These changes should improve the migration flow and make a few things more clear.

## What changed?
- The configs should be moved, before installing edumfa. Otherwise it will cause issues with deb package installation.
- The rename instructions where a bit broad. I made it more clear, what has to be renamed.
- I added a hint to not accept the new config during the edumfa installation, otherwise it may cause issues
- It wasn't pretty clear how to call the migration binary. I fixed that

### CODE REVIEW

File: READ_BEFORE_UPDATE.md
Comment (fritterhoff) [inline READ_BEFORE_UPDATE.md]: Is there a reason for the new line breaks? (Some formatting anywhere that requires a specific line length? 🤔 )

Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: This is the automated code format from Jetbrain IDEs

Comment (fritterhoff) [reply READ_BEFORE_UPDATE.md]: Would you mind to change your settings? There should be a setting for line breaks. Would prefer to keep the changes in existing changelogs as small as possible,

File: READ_BEFORE_UPDATE.md
Comment (fritterhoff) [inline READ_BEFORE_UPDATE.md]: Maybe it makes sense to shift this block to the top of the release notes and make it more generic? Migrating from privacyIDEA 3.9.2 to eduMFA

**Comment by Luc1412** [comment]
@fritterhoff Solved the line wrapping

**Review by fritterhoff — APPROVED** [review]

**Comment by fritterhoff** [comment]
@j-hoff since I changed some lines we need your approval. 😬

File: READ_BEFORE_UPDATE.md
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]:
```suggestion
* Uninstall privacyIDEA and stop your `Apache2` or `nginx` service
```

File: READ_BEFORE_UPDATE.md
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: how? will the config be overwritten in any case?

File: READ_BEFORE_UPDATE.md
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: I believe crontab or systemd services will not be updated by the .deb packages

File: READ_BEFORE_UPDATE.md
Comment (j-hoff) [inline READ_BEFORE_UPDATE.md]: maybe we change this line until we release a version for which the instructions do NOT work
```suggestion
> These migration steps are applicable for all versions released up to now
```

**Review by j-hoff — CHANGES_REQUESTED** [review]
some minor remarks

File: READ_BEFORE_UPDATE.md
Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: When having a config present, during installation you'll be asked if you like to keep the current config or use the maintainers version.

File: READ_BEFORE_UPDATE.md
Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: The packages adds a new cron, however systemd keeps unchanged. Maybe we should rather mention, what will be done automatically like: "When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will executed automatically. Additionally the web server configurtion and a cron will also be installed.

Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]:
```suggestion
    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will be executed automatically. Additionally the web server configuration and a cron entry will be installed.
```

File: READ_BEFORE_UPDATE.md
Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]:
```suggestion
    * When using the `.deb` package, make sure to not overwrite the existing configuration file (When asked by dpkg, answer N or O to keep your installed version)
```

Comment (Luc1412) [reply READ_BEFORE_UPDATE.md]: I think the current explanation should be mostly sufficient. Maybe we can add `during installation` and/or `when prompted`

Comment (j-hoff) [reply READ_BEFORE_UPDATE.md]: OK
```suggestion
    * When using the `.deb` package, make sure not to overwrite the existing configuration file when prompted
```

**Review by j-hoff — APPROVED** [review]

**Pull request merged**
